src/assets/config.json

```diff
@@ -34,6 +34,8 @@
   "addContainerCost": 1500,
   "holdingCostPerSize": 0.05,
   "costExtraTruck": 100,
+  "costSizeTruck": 50,
+  "sizeIncreasingPercentage" : 1.1,
```

Comment:
@lisavancasteren a bit nitpicky, but I still think using `factor` rather than `percentage` is better